addons/account/models/sequence_mixin.py

```diff
@@ -237,5 +253,12 @@ def _set_next_sequence(self):
             format_values['month'] = self[self._sequence_date_field].month
         format_values['seq'] = format_values['seq'] + 1
 
-        self[self._sequence_field] = format.format(**format_values)
-        self._compute_split_sequence()
+        return format.format(**format_values)
+
+    def _is_last_from_seq_chain(self): #TODO OCO un test pour ces trucs à la suppression sur les moves ? Ou en étoffer un ?
+        """ Tells whether or not this element is the last one of the sequence chain
+        matching its date.
+        :return: True if it's the last element of the chain
+        """
+        self.ensure_one()
+        return self[self._sequence_field] == self._generate_candidate_sequence(self[self._sequence_field])
```

Comment:
Because it doesn't work :p 

With your suggestion, we won't be able to delete the last element of the chain if there is a chain. If we have INV/2021/03/01, INV/2021/03/02 and INV/2021/03/03, calling _get_last_sequence on the second invoice will return INV/2021/03/03 and ... allow deletion ? We only want to authorize deletion for INV/2021/03/03 here. 

Actually, I had tried using get_last_sequence more directly, but always had tons of corner cases ... And trying to find other arguments, I realized this solution also has some ~~ I might have an idea, though. Working on it. And I'll add a test in the meantime.